components/suggest/src/schema.rs

```diff
@@ -28,6 +28,17 @@ CREATE TABLE meta(
     value NOT NULL
 ) WITHOUT ROWID;
 
+CREATE TABLE rs_cache(
+    collection TEXT PRIMARY KEY,
+    data TEXT NOT NULL
```

Comment:
Storing record id and mtime seems interesting, I'll consider that for #6328.  I switched the serialization format to msgpack, which seems decent enough.  I tried bincode and postcard, but neither of them can handle the `serde_json::Value` field.  I think that's because it can contain heterogeneous lists.